cfme/fixtures/service_fixtures.py

```diff
@@ -12,23 +12,43 @@
 from cfme.infrastructure.provider import InfraProvider
 from cfme.rest.gen_data import dialog as _dialog
```

Comment:
Question: Why are `catalog` and `dialog` not fixtures in the first place?

Reply:
Those are test sckeleton